documentation/docs/release-notes/3.1.0.md

```diff
@@ -49,6 +49,34 @@ This enhancement provides a more accurate and intuitive visualization of your Mo
 ![Topology section](Topology.png)
 
 
+### Extended metrics and labels for MongoDB Query Analytics (QAN)
+
+We've added new metrics and filters to QAN that expose important query execution details, highlighting inefficient execution paths, providing detailed lock and storage metrics. With the new filters, you can also easily identify these problematic queries by application, client, or user:
+
+
+#### Additional MongoDB QAN metrics
+The following metrics are now available in QAN for MongoDB queries:
+
+- `docs_examined`: number of documents scanned during query execution
+- `keys_examined`: number of index entries scanned
+- `nreturned`: number of documents returned by the query
+- `response_length`: size of the response in bytes
+- `millis`: query execution time in milliseconds
+- `locks_global_acquire_count_read_shared`: global read lock acquisition count
+- `locks_global_acquire_count_write_shared`: global write lock acquisition count
+- `locks_database_acquire_count_read_shared`: database read lock acquisition count
+- `locks_database_acquire_wait_count_read_shared`: database read lock wait count
+- `locks_database_time_acquiring_micros_read_shared`: time spent acquiring database locks in microseconds
+- `locks_collection_acquire_count_read_shared`: collection read lock acquisition count
+- `storage_bytes_read`: amount of data read from storage in bytes
+- `storage_time_reading_micros`: time spent reading from storage in microseconds
```

Comment:
Can we use for each metric it's humanized name from here https://github.com/percona/grafana-dashboards/pull/1656/files#diff-179300d38c32860677d134b1586267e655e28b77c0f7a06018f139f04b8a6925R1115?